.evergreen/generate_evergreen_tasks.js

```diff
@@ -580,83 +545,62 @@ BUILD_VARIANTS.push({
 const oneOffFuncs = [
   {
     name: 'run-custom-snappy-tests',
-    func: 'run custom snappy tests'
+    func: 'run custom snappy tests',
+    expansions: {}
```

Comment:
What is this change for?

Reply:
Originally I added this so it could be spread on line 570, but omitting it still works because `{ ...undefined }` => `{}`